### PR TITLE
PT-BR translation on ct.js-site

### PR DESCRIPTION
**Pull request by GumpFlash**
I added the PT-BR translation to ct.js-site, but i don't finish the changelog, btw the portuguese site it's there

### CODE REVIEW

File: src/br/brand.pug [hunk 1]
```diff
@@ -0,0 +1,14 @@
+extends pugTemplates/basic.pug
```
Comment (CosmoMyzrailGorynych) [inline src/br/brand.pug]: Omg, I thought I didn't commit these

File: src/br/contact.pug [hunk 2]
```diff
@@ -0,0 +1,29 @@
+extends pugTemplates/basic.pug
+block head
+    title Contato — Ct.js
+    meta(name="description" content="Envie um email para o desenvolvedor.")
```
Comment (CosmoMyzrailGorynych) [inline src/br/contact.pug]: I don't want contact forms to be translated for a couple of reasons:

* English forms suggest that I accept mail written in English;
* Several forms are harder to track on Netlfy and to automate with cross-posting.

Comment (GumpFlash) [reply src/br/contact.pug]: do you want i don't translated or place an advice, saying like 'please submit only english form'?

File: src/br/brand.pug (hunk 1 above)
Comment (GumpFlash) [reply src/br/brand.pug]: do you want me delete this file?

File: src/br/contact.pug (hunk 2 above)
Comment (CosmoMyzrailGorynych) [reply src/br/contact.pug]: The latter would be a good solution :+1:

File: src/br/brand.pug (hunk 1 above)
Comment (CosmoMyzrailGorynych) [reply src/br/brand.pug]: Idk, you can leave it be, it is still not displayed ¯\_(ツ)_/¯

**Comment by GumpFlash** [comment]
- npm-debug.log Added on .gitignore
- /_includes/whatsnext_br.pug created!
- advice in the portuguese contact page placed!

and i don't know who make this: 'I'll also try to add some js sorcery so that the header/footer links to specific translated pages only if they exist.', sorry :( 
i don't know glop ;(

**Comment by CosmoMyzrailGorynych** [comment]
So my sorcery failed and I added plain simple and boring variables to `locale.11tydata.js`. But it now allows more fine-grained control over linking.

**Pull request merged**